packages/cli/lib/validators/marketplaceValidators/themeConfigValidator.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+
+const { VALIDATION_RESULT } = require('../constants');
+
+const VALIDATOR_NAME = 'ThemeConfigValidator';
+const THEME_JSON_REGEX = new RegExp(/^theme\.json$/);
+
+// Validate that the theme contains a theme.json file
+function themeConfigValidator(absoluteThemePath, files) {
+  let validationErrors = [];
+  const themeJSONFile = files.find(fileName => {
+    return THEME_JSON_REGEX.test(fileName);
```

Comment:
Instead of a regex, could this be `fileName === 'theme.json'`?

Reply:
Yep! Much simpler. Done in ee4a6dd